run.sh

```diff
@@ -29,9 +29,6 @@ worker)
     echo "starting cron"
```

Comment:
We need to take this line out and the following one too

Reply:
Thanks, removed the ones above aswell